app/models/reaction.rb

```diff
@@ -19,9 +19,9 @@ class Reaction < ApplicationRecord
   validate  :permissions
 
   before_save :assign_points
-  after_commit :async_bust
```

Comment:
We need to leave this where it is rather than changing this behavior bc this also triggers a Sidekiq worker that needs to ensure the model has been committed to the db before running

Reply:
So it will be
```ruby
after_commit :async_bust
after_commit :bust_reactable_cache
after_save :update_reactable, :touch_user, :async_bust
```

Reply:
@mstruve ?

Reply:
We are only changing the `bust_reactable_cache` method therefore that is the only method that should be changing here. `async_bust` should not change and be added to the`after_save` callback

Reply:
Sure, sending the changes;